test_scripts/Smoke/Registration/ATF_Register_5_connection.lua

```diff
@@ -104,30 +96,23 @@ end
 --[[ Test ]]
 commonFunctions:newTestCasesGroup("Test")
 
-function Test:FirstConnection()
-  createConnectionAndRegisterApp(self, device1, "mobile1.out", default_app_params1)
-end
-
-function Test:SecondConnection()
-  createConnectionAndRegisterApp(self, device2, "mobile2.out", default_app_params2)
-end
-
-function Test:ThirdConnection()
-  createConnectionAndRegisterApp(self, device3, "mobile3.out", default_app_params3)
+for i = 1, 5 do
+  function Test:FirstConnection()
```

Comment:
@dboltovskyi  Please rename function in loop (`First Connection`) with appropriate name.